samtools/README.md

```diff
@@ -26,13 +34,83 @@ Scratch directory:
 ```
 
 ## (3) Push the Docker image to a repository.
+
 In this example, we push the container to [Google Container Registry](https://cloud.google.com/container-registry/) via the following commands:
 ```
 docker tag ${USER}/samtools gcr.io/YOUR-PROJECT-ID/samtools
 gcloud docker push gcr.io/YOUR-PROJECT-ID/samtools
 ```
 
-## (4) Run the Docker image in the cloud
+## (4) Run the Docker image in the cloud, using gcloud
+
+The `gcloud` tool that comes with the Google Cloud SDK includes a command
+to run pipelines. You can get details of the command with:
+
+```
+gcloud alpha genomics pipelines run --help
+```
+
+### (4a) Run the Docker image in the cloud, using gcloud
+
+To run this example, first edit the included [./cloud/samtools.yaml](./cloud/samtools.yaml) file:
+
+* Replace `YOUR-PROJECT-ID` with your project ID.
+
+### (4b) Execute the `pipelines run` command:
+
+```
+gcloud alpha genomics pipelines run \
+  --pipeline-file cloud/samtools.yaml \
+  --inputs inputPath=gs://genomics-public-data/ftp-trace.ncbi.nih.gov/1000genomes/ftp/technical/pilot3_exon_targetted_GRCh37_bams/data/NA06986/alignment/NA06986.chromMT.ILLUMINA.bwa.CEU.exon_targetted.20100311.bam \
+  --outputs outputPath=gs://YOUR-BUCKET/pipelines-api-examples/samtools/gcloud/output/ \
+  --logging gs://YOUR-BUCKET/pipelines-api-examples/samtools/gcloud/logging/ \
+  --disk-size datadisk:100
+Running: [operations/YOUR-NEW-OPERATION-ID]
+```
+
+* Replace `YOUR-BUCKET` with a bucket in your project.
+
+### (4c) Monitor the pipeline operation
+
+This github repo includes a shell script, [../tools/poll.sh](../tools/poll.sh), for monitoring the completion status of an operation.
+
+```
+$../tools/poll.sh YOUR-NEW-OPERATION-ID
+Operation not complete. Sleeping 20 seconds
+Operation not complete. Sleeping 20 seconds
+...
+Operation not complete. Sleeping 20 seconds
+
+Operation complete
+done: true
+metadata:
+  '@type': type.googleapis.com/google.genomics.v1.OperationMetadata
+  clientId: ''
+  createTime: '2016-05-04T17:21:07.000Z'
+  endTime: '2016-05-04T17:22:34.000Z'
+...
+  startTime: '2016-05-04T17:21:36.000Z'
+name: operations/YOUR-NEW-OPERATION-ID
+
+```
+
+### (4d) Check the results
+
+Check the operation output for a top-level `errors` field.
+If none, then the operation should have finished successfully.
+
+```
+$ gsutil ls gs://YOUR-BUCKET/pipelines-api-examples/samtools/gcloud/output
+gs://YOUR-BUCKET/pipelines-api-examples/samtools/gcloud/output/NA06986.chromMT.ILLUMINA.bwa.CEU.exon_targetted.20100311.bam.bai
+```
+
+## (5) Run the Docker image in the cloud, using the Python client libraries
```

Comment:
Perhaps `(5) OPTIONAL: instead of gcloud, use the Python client libraries to run the Docker image in the cloud.`